package.json

```diff
@@ -96,7 +96,8 @@
     "test-help": "echo For development it is recommended to run yarn test-prepare-watch in one terminal and yarn test-watch in another. This is the fastest way to perform incremental testing. If you are only interested in running test once, use yarn test.",
     "remove-e2e-snapshots": "rm -rf frontend/javascripts/test/snapshots/public-test/test-bundle/test/backend-snapshot-tests/ && rm -rf frontend/javascripts/test/snapshots/public-test/test-bundle/test/enzyme/",
     "remove-all-snapshots": "rm -rf frontend/javascripts/test/snapshots/",
-    "refresh-e2e-snapshots": "yarn remove-e2e-snapshots && mkdir -p frontend/javascripts/test/snapshots/type-check && docker compose down && docker compose up e2e-tests",
+    "clear-cmake-cache": "rm -f webknossos-jni/target/native/x86_64-linux/build/CMakeCache.txt",
+    "refresh-e2e-snapshots": "yarn remove-e2e-snapshots && mkdir -p frontend/javascripts/test/snapshots/type-check && yarn clear-cmake-cache && docker compose down && docker compose up e2e-tests && yarn clear-cmake-cache",
```

Comment:
why is the cmake cache clared when refreshing the e2e snapshots? ideally, the docker image wouldn't change `webknossos-jni`. can we "unmount" it?

Reply:
I now see that @frcroth also wrote an issue for this: https://github.com/scalableminds/webknossos/issues/7707

Reply:
ok, makes sense :+1: not sure how much time the cache usually saves, but an alternative would be to rename the cache to `*.bak*` before refreshing the snapshots and afterwards renaming it back (thus, overwriting the cache that was created in between).

Reply:
Good point! I changed it to do just that now :)